src/Context/LoginContext.php

```diff
@@ -80,6 +81,48 @@ public function stepIAmNotLoggedIn()
      * @param string $password
      */
     public function stepILogInWith($email, $password)
+    {
+        $this->loginWith($email, $password);
+
+        // Check if MFA module is installed
+        if (!class_exists(RegisteredMethod::class)) {
+            return;
+        }
+
+        // Skip MFA registration if MFA module installed
+        $this->getMainContext()->getSession()->wait(300);
+        $page = $this->getMainContext()->getSession()->getPage();
+        $mfa = $page->find('css', '#mfa-app');
+        if (!$mfa) {
+            return;
+        }
+        $clicked = false;
+        $btns = $page->findAll('css', '.mfa-action-list__item .btn');
+        foreach ($btns as $btn) {
+            if ($btn->getText() !== 'Setup later') {
+                continue;
+            }
+            $btn->click();
+            $clicked = true;
```

Comment:
You should probably `break;` here,  otherwise you'll keep looping buttons.

Reply:
You'd think so, though for some reason the button isn't actually clicked when adding a break here.  I suspect there's some weird intricacy within behat/mink.  Makes zero practical different not having a break here since the button we click is the last button.

Reply:
Updated